PriceCalculatorKata/Expense.cs

```diff
@@ -0,0 +1,35 @@
+﻿using PriceCalculatorKata.Common;
+
+namespace PriceCalculatorKata;
+
+public struct Expense
+{
+    private string? _name;
```

Comment:
this is a potential uninitialized variable exception, it should be initialized to null for instance to avoid that